tests/e2e/test_operator.py

```diff
@@ -2142,6 +2142,54 @@ def test_032(self):
     kubectl.delete_chi(chi)
     kubectl.launch(f"delete pod clickhouse-test-032-client", ns=kubectl.namespace, timeout=600)
 
+
+@TestScenario
+@Name("test_033. Check HTTPS support for health check")
+def test_033(self):
+    def check_monitoring_metrics(operator_namespace, operator_pod, expect_result, max_retries=10):
```

Comment:
instead of copy paste `check_monitoring_metrics` from test_metrics_exporter
you could make refactoring and move this function to `utils.py` and use utils.check_monitoring_metrics